.vscode/settings.json

```
@@ -1,8 +1,23 @@
 {
+    // formatting style this project adheres to: https://en.wikipedia.org/wiki/Indentation_style#Allman
+    "powershell.codeFormatting.preset": "Allman",
+    "powershell.codeFormatting.openBraceOnSameLine": false,

```

```suggestion
    // formatting style this project adheres to: https://en.wikipedia.org/wiki/Indentation_style#Stroustrup
    "powershell.codeFormatting.preset": "Stroustrup",
```
I prefer Stroustrup.